staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -22,15 +22,39 @@ import (
 	"encoding/asn1"
 	"fmt"
 	"net/http"
+	"time"
 
 	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 	"k8s.io/apiserver/pkg/authentication/user"
 )
 
+var clientCertificateExpirationHistogram = prometheus.NewHistogram(
```

Comment:
`:=`

Unless there's some obvious reason for `var ... =` that I'm not seeing...